packages/fxa-settings/Gruntfile.js

```diff
@@ -17,6 +12,16 @@ module.exports = function (grunt) {
 
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
+    // make a copy of the local branding terms available for local development
+    // before they are extracted to the l10n repo
+    // this file will not be included in the string extraction process, so should not lead to duplication
+    copy: {
+      'branding-ftl': {
+        nonull: true,
```

Comment:
What's this do?

Reply:
From the docs: 
> By default, if a file or directory is not found it is quietly ignored. If the file should exist, and non-existence generate an error, then add nonull:true

Reply:
Almost wonder if we should be using this in more places then. 🤔